tests/models/deberta/test_modeling_deberta.py

```diff
@@ -198,6 +200,31 @@ def create_and_check_deberta_for_question_answering(
         self.parent.assertEqual(result.start_logits.shape, (self.batch_size, self.seq_length))
         self.parent.assertEqual(result.end_logits.shape, (self.batch_size, self.seq_length))
 
+    def create_and_check_deberta_for_tracing_output_validity(
+        self, config, input_ids, token_type_ids, input_mask, sequence_labels, token_labels, choice_labels
+    ):
+        config_trace = deepcopy(config)
+        config_trace.torchscript = True
+
+        model = DebertaModel(config=config_trace)
+        model.to(torch_device)
+        model.eval()
+        sequence_output = model(input_ids, attention_mask=input_mask, token_type_ids=token_type_ids)
+        sequence_output = sequence_output[0].flatten().tolist()
+
+        trace_input = (input_ids, input_mask, token_type_ids)
+        traced_model_data = torch.jit.trace(model, trace_input)
+        model_bytes = io.BytesIO()
+        torch.jit.save(traced_model_data, model_bytes)
+        model_bytes.seek(0)
+
+        traced_model = torch.jit.load(model_bytes)
+        traced_model.eval()
+        traced_sequence_output = traced_model(input_ids, attention_mask=input_mask, token_type_ids=token_type_ids)
+        traced_sequence_output = traced_sequence_output[0].flatten().tolist()
+
+        self.parent.assertListEqual(sequence_output, traced_sequence_output)
+
     def prepare_config_and_inputs_for_common(self):
         config_and_inputs = self.prepare_config_and_inputs()
         (
```

Comment:
We have a `    test_torchscript = False` attribute line 243 of the test file that should be set to true!